acceptance/redis32/redis-ha.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+set -xe
+
+# Test Redis is working by running `SET`, `GET`, and `DEL`.
+run_tests() {
```

Comment:
Do we need to run this test here? It's already getting run via `run-acceptance-test.sh`.